src/test/java/de/tum/cit/aet/artemis/programming/icl/ProgrammingExerciseLocalVCLocalCIIntegrationTest.java

```diff
@@ -291,7 +294,9 @@ void testImportProgrammingExercise() throws Exception {
         var params = new LinkedMultiValueMap<String, String>();
         params.add("recreateBuildPlans", "true");
         exerciseToBeImported.setChannelName("testchannel-pe-imported");
-        exerciseToBeImported.setCompetencies(Set.of(competency));
+        exerciseToBeImported.setCompetencyLinks(Set.of(new CompetencyExerciseLink(competency, exerciseToBeImported, 1)));
+        exerciseToBeImported.getCompetencyLinks().forEach(link -> link.getCompetency().setCourse(null));
```

Comment:
Does leaving in the course lead to a problem?

Reply:
This leads to problems since the course again contains the competency and we get circular dependencies